flash_update/serializers.py

```diff
@@ -59,6 +58,8 @@ class Meta:
         read_only_fields = ('created_by',)
 
     def create(self, validated_data):
+        if 'file' in validated_data:
+            check(validated_data['file'])
```

Comment:
For create, We can ignore "file" in validated_data check as the `file is a required field in FlashGraphicMap.